apps/src/templates/curriculumCatalog/expanded_curriculum_catalog_card.module.scss

```diff
@@ -36,5 +117,68 @@ $container-width: 960px;
 .infoContainer {
   display: flex;
   align-items: center;
-  column-gap: 1em;
+  column-gap: 20px;
+}
+
+.linksContainer {
+  hr {
+    margin: 8px 0;
+  }
+
+  i {
+    margin-left: 8px;
```

Comment:
Nit: try to avoid `-left` or `-right` since right-to-left-reading languages might style weird. I believe `margin-inline-start` will do the same thing here but I'm not 100% sure